datafusion/core/tests/sqllogictests/test_files/order.slt

```diff
@@ -155,6 +154,54 @@ SELECT c1, c2 FROM test ORDER BY c1 DESC, c2 ASC
 0 9
 0 10
 
+# eliminate duplicated sorted expr
```

Comment:
👍  it would also be awesome to add an `EXPLAIN` test to show that the sort was only on `c1, c2`